src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,51 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
     public static void main(String[] args) {
-        // test your code here
+
+        Driver johnLennon = new Driver("John Lennon", "S123-456-600-12");
+        Driver polMccartney = new Driver("Pol Mccartney", "K200-546-176-01");
+        Driver georgeHarrison = new Driver("George Harrison", "W414-842-763-22");
+        Driver ringoStarr = new Driver("Ringo Starr", "A134-746-978-75");
+        Driver stuartSutcliffe = new Driver(4L,"Stuart Sutcliffe", "F22-11-33-34");
+        driverService.create(johnLennon);
+        driverService.create(polMccartney);
+        driverService.create(georgeHarrison);
+        driverService.create(ringoStarr);
+
+        driverService.delete(1L);
+        driverService.update(stuartSutcliffe);
+        System.out.println(driverService.get(2L));
+        System.out.println(driverService.getAll());
```

Comment:
```suggestion
        Driver johnLennon = new Driver("John Lennon", "S123-456-600-12");
        Driver polMccartney = new Driver("Pol Mccartney", "K200-546-176-01");
        Driver georgeHarrison = new Driver("George Harrison", "W414-842-763-22");
        Driver ringoStarr = new Driver("Ringo Starr", "A134-746-978-75");
        Driver stuartSutcliffe = new Driver(4L,"Stuart Sutcliffe", "F22-11-33-34");
        johnLennon = driverService.create(johnLennon);
        polMccartney = driverService.create(polMccartney);
        georgeHarrison = driverService.create(georgeHarrison);
        ringoStarr = driverService.create(ringoStarr);

        driverService.delete(ringoStarr.getId());
        georgeHarrison.setLicenseNumber("newLicence")
        driverService.update(georgeHarrison);
        System.out.println(driverService.get(georgeHarrison.getId()));
        System.out.println(driverService.getAll());
```

it's better to test that way